src/main/kotlin/chapter5/Listing_5_3.kt

```diff
@@ -45,32 +45,35 @@ object Listing_5_3 {
     val trace = {
         //tag::trace[]
         Stream.of(1, 2, 3, 4).map { it + 10 }
-            .filter { it % 2 == 0 }.toList()
+            .filter { it % 2 == 0 }.map { it * 3 }.toList()
 
         cons({ 11 }, { Stream.of(2, 3, 4) })
-            .filter { it % 2 == 0 }.toList() // <1>
+            .filter { it % 2 == 0 }.map { it * 3 }.toList() // <1>
 
         Stream.of(2, 3, 4).map { it + 10 }
-            .filter { it % 2 == 0 }.toList() // <2>
+            .filter { it % 2 == 0 }.map { it * 3 }.toList() // <2>
 
         cons({ 12 }, { Stream.of(3, 4) })
-            .filter { it % 2 == 0 }.toList() // <3>
+            .filter { it % 2 == 0 }.map { it * 3 }.toList() // <3>
 
-        ConsL(12, Stream.of(3, 4).map { it + 10 }
-            .filter { it % 2 == 0 }.toList()) // <4>
+        ConsL(36, Stream.of(3, 4).map { it + 10 }
+            .filter { it % 2 == 0 }.map { it * 3 }.toList()) // <4>
 
         ConsL(
```

Comment:
I see we have some strange formatting on this one iteration. Could you fix it as part of this PR?